src/google/google-calendars/googlecalendarsyncadaptor.cpp

```diff
@@ -592,6 +598,40 @@ void GoogleCalendarSyncAdaptor::finalCleanup()
     if (succeededAccounts.size()) {
         setLastSyncSuccessful(succeededAccounts);
     }
+
+    // Delete any events which are not associated with a notebook.
+    // These events are ghost events, caused by a bug which previously
+    // existed in the sync adapter code due to mkcal deleteNotebook semantics.
```

Comment:
Has the bug been in some released version? I understood some ghost events were resulting from earlier version of this PR.